### DISCUSSION TITLE
Mandate 2FA for verified students and teachers

### DISCUSSION THREAD

**Discussion by thispsj** [community/community#19155] [GitHub Education] [answered]
Hi GitHub Education Team,

Given the amount of digital freebies given to verified teachers and students among other benefits, their accounts maybe at risk of getting compromised hence you should make 2FA mandatory for all verified students and teachers before they can claim their Student/Teacher pack.

Just an idea.

Regards,
@thispsj

**Comment by samus-aran** [comment, marked as answer]
Thanks @thispsj for this feedback. GitHub are applying mandatory 2FA for accounts that [contribute code on GitHub.com by 2023](https://github.blog/2022-05-04-software-security-starts-with-the-developer-securing-developer-accounts-with-2fa/). Whether the parameter of "contributes to code" applies to students & teachers upon account creation or only when creating, merging pull requests could be clarified more.

**Comment by ghost** [comment]
https://en.wikipedia.org/wiki/Collateral_damage
